migrations/20220801135839_answer_count_and_timestamps.js

```diff
@@ -4,13 +4,13 @@
  */
 exports.up = async function (knex) {
   await knex.schema.table("questions", (table) => {
-    table.time("created_at");
+    table.time("timestamp");
```

Comment:
This is not really how migrations work. One is meant to never change an existing migration but instead create new migrations that alter the schema. That's the whole point of the migration.

Reply:
my bad I wanted to "clean up" my migrations before pushing I should not have done that